### DISCUSSION TITLE
Opt-out of Pro before billing begins?

### DISCUSSION THREAD

**Discussion by KDLGates** [AntlerVC/firetable#874] [Support / Q&A] [answered]
Sorry for the weak question, I used Rowy for a Hackathon project and honestly enjoyed it, but upgraded to Pro without realizing it will transition to a paid feature
Is there a way for me to opt out of Pro or not worry about it? New to such things and unsure if it might be ready to start auto-billing my Google account tied to the Firebase instance or similar to be honest, just concerned about the possibility of an unexpected charge when I don't expect to use Pro enough for it to be worth.
Was expecting to find an account opt-out but I guess it's still in free testing so that might not be a thing yet.

**Comment by harinij** [comment, marked as answer]
As discussed on Discord chat, currently Rowy is free and you will not incur any bill from Rowy end. We will be introducing Pro option to upgrade on Rowy soon which you can opt in or out at that point. On the GCP, you can turn on/off your Google cloud billing.
